Makefile

```diff
@@ -51,23 +51,27 @@ $(KUBEBUILDER_ASSETS):
 
 .PHONY: unit-tests
 unit-tests::install-tools ## Run unit tests
-unit-test::$(KUBEBUILDER_ASSETS)
-unit-test::generate
-unit-test::fmt
-unit-test::vet
-unit-test::vuln
```

Comment:
is vuln removed now?

Reply:
I split in two commits the code changes and the Makefile: f13f3597960e9240e99c1d71ec300385f905ba0e

In summary, I don't think there's much value in running `vuln` during unit tests. The `vuln` scanner still is available to run manually at any time.